packages/puppeteer-core/src/node/FirefoxLauncher.ts

```diff
@@ -113,7 +113,20 @@ export class FirefoxLauncher extends ProductLauncher {
 
     await createProfile(SupportedBrowsers.FIREFOX, {
       path: userDataDir,
-      preferences: extraPrefsFirefox,
+      preferences: {
+        ...extraPrefsFirefox,
+        ...(options.protocol === 'cdp'
+          ? {
+              // Temporarily force disable BFCache in parent (https://bit.ly/bug-1732263)
+              'fission.bfcacheInParent': false,
```

Comment:
We should still be able to get rid of this preference given that it doesn't affect the isolation feature.

Reply:
Isn't required for the CDP version?

Reply:
Oh sorry, yes. With the usage of `? :` it is quite hard to read the code here in the diff viewer. So for CDP we need both preferences set, and for BiDi only `'fission.webContentIsolationStrategy': 0`.

Reply:
If you need a reference you can use https://bugzilla.mozilla.org/show_bug.cgi?id=1773393.

Reply:
Thanks!